### ISSUE TITLE
Accessibility issue with social media links

### ISSUE THREAD

**Issue by hiwelo** [Rudloff/alltube#212]
To be accessible (WCAG: 2.4.4, 4.1.2), links *must have* discernible text.
Two links on the homepage are failing to meet the requirement:

- [ ] Facebook
- [ ] Twitter

Both links are considered empty because there is no content and the `title` attribute is not a discernible text.
You can fix it by adding an `aria-label` or a screen-reader only text.
